File: 백준/Bronze/1546. 평균/평균.cc
#include <iostream>
#include <vector>
using namespace std;

int main(void)
{
	int N;
	double sum = 0;
	vector<int> A;

	cin >> N;

	for (int i = 0; i < N; i++)
	{
		int data;
		cin >> data;
		A.push_back(data);
	}

	int max = A[0];
	for (int j = 0; j < N; j++)
	{
		if (max < A[j])
			max = A[j];
		sum += A[j];
	}

	cout << sum * 100 / max / N;

	return 0;
}

File: 백준/Bronze/1546. 평균/평균.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int[] A = new int[N];

        for (int i = 0; i < N; i++) {
            A[i] = sc.nextInt();
        }

        double sum = 0;
        double max = 0;
        for (int i = 0; i < N; i++) {
            if (A[i] > max) {
                max = A[i];
            }
            sum += A[i];
        }

        System.out.println(sum * 100 / max / N);
    }
}